Automattic/wp-calypso
# Signup: Update Site Title step reducer name

In #8652 a new naming scheme, explicitly specifying the action, was proposed for Signup reducers. 

This PR updates the Site Title reducer to follow that naming scheme.

There shouldn't be any behavioral changes with this change.

To test:

1. Checkout branch or use Calypso.live branch
2. Start signup
3. Make sure you're using `showSiteTitleStep` variation for `siteTitleStep`
4. Go through signup
5. See if the site title you wrote in Signup is applied to the newly created site. 
6. Make sure there aren't any errors in the console. 

cc @coreh @michaeldcain
